resources/views/dashboard/controllers/bookings/view_bookings.blade.php

```diff
@@ -0,0 +1,109 @@
+@extends('layouts.dashboard')
+
+@section('title')
+ATC Bookings
+@endsection
+
+@section('content')
+@include('inc.header', ['title' => 'ATC Bookings'])
+
+<div class="row">
+    <div class="col-lg-6">
+        @if(count($bookings) == 0)
+            <div class="card mb-3">
+                <div class="card-body text-center">
+                    <h4>No ATC Bookings</h4>
+                    <p>There are no ATC Bookings at this time. Create the first one using the form to the right!</p>
+                    <h4 class="fa fa-clock"></h4>
+                </div>
+            </div>
+        @else
+            @foreach($bookings as $date => $date_bookings)
+                <table class="table">
+                    <thead class="thead-light">
+                        <tr>
+                            <th colspan="2">{{ $date }}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach($date_bookings as $b)
+                            <tr>
+                                <td>
+                                    <strong>{{ $b->callsign }}</strong>
+                                    &emsp;
+                                    <span>{{ $b->start_time_formatted }} - {{ $b->end_time_formatted }}</span>
+                                    @if($b->cid == Auth::id() || Auth::user()->can('snrStaff'))
+                                        <span class="float-right"><a href="/dashboard/controllers/bookings/delete/{{ $b->id }}" class="btn btn-sm btn-danger">Cancel</a></span>
+                                    @endif
+                                </td>
+                            </tr>
+                        @endforeach
+                    </tbody>
+                </table>
+            @endforeach
+        @endif
+        <small><i>Only bookings you create are visible beyond 14 days from today</i></small>
```

Comment:
What does this mean? Unclear based on the verbiage used. Can you clarify?

Reply:
It is saying that there may be bookings beyond the next 14 days that aren't showing because they aren't yours. Any future bookings created by a particular controller will show, but bookings by others will only show within the following 14 days. I struggled wording this so if you have a recommendation to make it clearer we should probably change it

Reply:
Ok, gotcha. Maybe something like "All of your bookings are listed above, but you can only see bookings made by others 14 days in advance."

Reply:
Resolved and squashed into b265bba